docs/src/modules/javascript/pages/developer-tools.adoc

```diff
@@ -0,0 +1,64 @@
+= Developer Tools
+
+The Javascript tooling is published to npm or yarn under the https://www.npmjs.com/org/lightbend[`@lightbend` organisation].
+
+== Kickstart
+
+Entity initialization is provided via the _create-akkasls-entity_ command line tool. This tool requires only a name for the new entity, and creates a new NPM project directory with the Akka Serverless SDK and associated development support tooling set up.
```

Comment:
Reminds me that `create-akkasls-entity` should probably become `create-akkasls-component` or similar, when it can generate actions and views.

Reply:
```suggestion
Entity initialization is provided via the _create-akkasls-entity_ command line tool. This tool requires only a name for the new entity, and creates a new npm project directory with the Akka Serverless SDK and associated development support tooling set up.
```

Reply:
> Reminds me that create-akkasls-entity should probably become create-akkasls-component or similar, when it can generate actions and views.

Agreed!